main.js

```diff
@@ -3,4 +3,6 @@ require('babel-register')({
     presets: ['env']
 });
```

Comment:
You might want to move this into a .babelrc config:
Use babel-node for development and babel -d for builds: 
Take a look at engine repo .babelrc and package.json https://github.com/Project-Rekt/engine

Reply:
@KingJeremyNg I think this one is for you